### ISSUE TITLE
Meet w/ SPL to discuss Special Campaigns

### ISSUE THREAD

**Issue by blaughman** [usdoj-crt/crt-portal-management#1400]
## User story:
As a CRT user, I want to create a special campaign page to get fewer, better reports.

## Objective:
Targeted intake for open investigations

## Key result:
Fewer, better reports

## Ticket background:
We have researched special campaigns and also have wireframes ready to be developed.  We need to get input from SPL on the language and scope of this work.

**Comment by blaughman** [comment]
Meeting set for 10/6 @ 10:30.

**Comment by randyabramson** [comment]
To recap this meeting:

1. We reviewed the current campaigns page: https://www.justice.gov/crt/special-litigation-section-case-summaries/download#Phoenix

2. We realized that updating and translating this page on our site would be problematic

3. We also realized that the page mentioned itself is probably not findable or is navigable/friendly. We also noted that this page will port to the new design in December.

4. Next steps:

a. Pilot pre-filled links that go from https://www.justice.gov/crt/special-litigation-section-case-summaries/download#Phoenix to Portal and measure the number of visits + reports completed

b. If it is low, change attention to redesiging this page: https://www.justice.gov/crt/special-litigation-section-case-summaries/download#Phoenix

**Comment by randyabramson** [comment]
@blaughman, after talking with Imra, about this project, a few issues emerged:

1. Tracking reports completed from a referral URL may be difficult because users might come to a report, go to another page and then come back. What EF tested was just tracking a referral url.

2. If the above is true, does the customer see any value in knowing the number of clicks from the SPL site to our site?

3. Beyond that, with Tag Manager, we may be able to track the full referral visit all the way through the form submission. If that's the case, we should look at our privacy policy.

In short, let's have a group conversation about this starting with the goal of the project.

**Comment by blaughman** [comment]
Closing this card.  We will create other cards to tackle the associated tasks with developing this feature.
